docs-v2/mint.json

```diff
@@ -169,7 +169,8 @@
                             "pages": [
                                 "customize/guides/advanced/handle-rate-limits",
                                 "customize/guides/advanced/handle-large-datasets",
-                                "customize/guides/advanced/migrate-integration-configuration"
+                                "customize/guides/advanced/migrate-integration-configuration",
+                                "customize/guides/advanced/validate-input-and-output"
```

Comment:
Should inputs & ouputs be plural?

Reply:
I searched while writing, they say it's accepted when you it's not countable and without s is the most common form. Happy to get @khaliqgant feedback on this